base/tps/src/org/dogtagpki/server/tps/TPSTokenPolicy.java

```diff
@@ -92,6 +105,41 @@ else if (policy[0].equalsIgnoreCase("RENEW_KEEP_OLD_ENC_CERTS"))
         }
     }
 
+    /* Take the current state of the policyt variables, create a new policy string,
+     * and write the new value for the provided token cuid.
+     */
+    public void updatePolicySet()  throws TPSException {
+   
+        String method = "TPSTokenPolicy.updatePolicySetString: "; 
```

Comment:
Did you intend to name the method "updatePolicySetString" instead?

Reply:
OH, I meant to change the method name, since the string seemed redundant. I can fix it either way you like. Change the acutal method name or change the method String that goes in the logs.